### DISCUSSION TITLE
GitHub is useless for Diff reviews with large numbers of comments and Suggestions.

### DISCUSSION THREAD

**Discussion by rchrdgrace** [community/community#72295] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

For coders or tech writers dealing with long docs in repos or large code files, GitHub is virtually **useless** for extensive reviews, **particularly when using Suggestions** for resolving items. **Google Doc is a more effective reviewing tool** than GitHub. There. I said it!

Since GH doesn't load large diffs in PR reviews by default, what happens when you have a file with a substantial number of Suggestions? You get to resolve one Suggestion at a time, have your diff refresh, and force you to re-open the diff and **manually scroll down again** to the next Suggestion. Repeat ad nauseum.

The PR review process for large docs is thus completely broken. There isn't a switch to enable this. You can't 'fix' more than one suggestion at a time. So every time you do this you're hitting the server with another re-load of the diff from the repo.  Yeah, I recognize you can't have changes get added to an outdated diff. But how about having a checklist of all Suggestions that allows for a one-shot process? Aren't these folks supposed to be the smartest coders in the tech industry? How about solving an actual problem users face for once?

Is there ever going to be a point where Microsoft actually cares about the GitHub user experience?

**Comment by ntc2** [comment]
Are you using Firefox on Linux? I have this problem you describe, when using Firefox on Linux, and it makes reviewing PRs on GitHub super painful as you describe. However, **it appears to be a bug in GitHub specific to Firefox for me: if I try Google Chrome, then the problem goes away**. So, whatever platform and browser you're on, if you aren't using Google Chrome, you might want to try that as a work around.

Before realizing this problem was due to the interaction of GitHub with Firefox, I was suggesting my team switch to GitLab if possible, since it doesn't have this problem, and the PR review process is better anyway (there's a "next PR comment" button, which is usually what I want).
